feature-libs/estimated-delivery-date/show-estimated-delivery-date/components/estimated-delivery-date.component.html

```diff
@@ -21,19 +21,7 @@
           {{ arrivalSlot?.at | cxDate: 'M/d/yyyy' }}
         </div>
         <ng-container *ngIf="arrivalSlot?.quantity">
-          <div
-            *cxFeature="'a11yQTY2Quantity'"
-            class="cx-value"
-            aria-hidden="true"
-          >
-            {{ 'estimatedDeliveryDate.quantity' | cxTranslate }}
-            {{ arrivalSlot?.quantity }}
-          </div>
-          <div
-            *cxFeature="'a11yQTY2Quantity'"
-            class="cx-value"
-            aria-hidden="true"
-          >
+          <div class="cx-value" aria-hidden="true">
             {{ 'estimatedDeliveryDate.quantityFull' | cxTranslate }}
             {{ arrivalSlot?.quantity }}
           </div>
```

Comment:
Looks like you removed the second div by mistake
```suggestion
       <ng-container *ngIf="arrivalSlot?.quantity">
         <div
            class="cx-value"
            aria-hidden="true"
          >
            {{ 'estimatedDeliveryDate.quantity' | cxTranslate }}
            {{ arrivalSlot?.quantity }}
          </div>
          <div
            class="cx-value"
            aria-hidden="true"
          >
            {{ 'estimatedDeliveryDate.quantityFull' | cxTranslate }}
            {{ arrivalSlot?.quantity }}
          </div>
```

Reply:
However, in `feature-libs/pickup-in-store/components/container/pickup-items-details/pickup-items-details.component.html` it looks very similar but with negation. @petarmarkov9449 how it should be?

Reply:
For me it looks like mistake. It should be one div with *cxFeature="'a11yQTY2Quantity'" and one div with *cxFeature="'!a11yQTY2Quantity'"
And in this case, looking on other components, I guess that this one is correct, and this should stay, only *cxFeature removed. and the second div should be removed (with 'cartItems.quantity'). but i'm guessing here
```
   <span
                    *cxFeature="'a11yQTY2Quantity'"
                    class="cx-label"
                    [ngClass]="'d-md-none d-lg-none d-xl-none'"
                    placement="left"
                    title="{{ 'cartItems.quantityTitle' | cxTranslate }}"
                  >
                    {{ 'cartItems.quantityFull' | cxTranslate }}
  </span>
```

Reply:
Just only the content changes from `quantity` to `quantityFull` based on the featureflag `a11yQTY2Quantity`.

Reply:
For this part: 
```
<div
          *cxFeature="'a11yQTY2Quantity'"
          class="cx-quantity-header col-md-1"
>
  {{ 'cartItems.quantityFull' | cxTranslate }}
</div>
<div
  *cxFeature="'!a11yQTY2Quantity'"
  class="cx-quantity-header col-md-1"
>
  {{ 'cartItems.quantity' | cxTranslate }}
</div>
```
must be
```
<div
  class="cx-quantity-header col-md-1"
>
  {{ 'cartItems.quantityFull' | cxTranslate }}
</div>

Reply:
And for this part 
```
<span
  *cxFeature="'a11yQTY2Quantity'"
  class="cx-label"
  [ngClass]="'d-md-none d-lg-none d-xl-none'"
  placement="left"
  title="{{ 'cartItems.quantityTitle' | cxTranslate }}"
>
  {{ 'cartItems.quantityFull' | cxTranslate }}
</span>
<span
  *cxFeature="'!a11yQTY2Quantity'"
  class="cx-label"
  [ngClass]="'d-md-none d-lg-none d-xl-none'"
  placement="left"
  title="{{ 'cartItems.quantityTitle' | cxTranslate }}"
>
      {{ 'cartItems.quantity' | cxTranslate }}
</span>
```
must be 
```
<span
      class="cx-label"
      [ngClass]="'d-md-none d-lg-none d-xl-none'"
      placement="left"
      title="{{ 'cartItems.quantityTitle' | cxTranslate }}"
    >
      {{ 'cartItems.quantityFull' | cxTranslate }}
</span>
```

Reply:
So your current work is good.